CHOCO_README.md

```diff
@@ -29,8 +29,8 @@ A command line interface to interact with [Rainforest QA](https://www.rainforest
 - `--import-variable-name NAME` - Use with `run` and `--import-variable-csv-file` to upload new tabular variable values before your run to specify the name of your tabular variable.
 - `--single-use` - Use with `run` or `csv-upload` to flag your variable upload as `single-use`. See `--import-variable-csv-file` and `--import-variable-name` options as well.
 - `--disable-telemetry` stops the cli sharing information about which CI system you may be using, and where you host your git repo (i.e. your git remote). Rainforest uses this to better integrate with CI tooling, and code hosting companies, it is not sold or shared. Disabling this may affect your Rainforest experience.
-- `--max-reruns` - If set to a value > 0 and a test fails, the CLI will re-run failed tests a number of times before reporting failure. If `--junit-file <filename>` is also used, the JUnit reports of reruns will be saved under `<filename>.1`, `<filename>.2` etc.
+- `--max-reruns` - If set to a value greater than 0 and a test fails, the CLI will re-run failed tests a number of times before reporting failure. If `--junit-file [filename]` is also used, the JUnit reports of reruns will be saved under `[filename].1`, `[filename].2` etc.
 
 ## Support
 
-Email [help@rainforestqa.com](mailto:help@rainforestqa.com) if you're having trouble using the CLI or need help with integrating Rainforest in your CI or development workflow.
+Email `help[at]rainforestqa.com` if you're having trouble using the CLI or need help with integrating Rainforest in your CI or development workflow.
```

Comment:
Did we run this through the validator? Is it obfuscated enough? Or is it just doing basic email validation?

Reply:
The current GitHub config pushes to Chocolatey on pushes to all branches (which I'll fix because that seems unideal), but we can see that the most recent version (with the changes from this PR) [was approved](https://community.chocolatey.org/packages/rainforest-cli#versionhistory):

> Rainforest CLI 3.1.0	Monday, September 26, 2022	Approved